### DISCUSSION TITLE
How in doughnut ChartWidget remove values/scales at x and y-coordinates

### DISCUSSION THREAD

**Discussion by PetroGromovo** [filamentphp/filament#9693] [Help] [answered]
I try to customize my doughnut ChartWidget reading docs :
https://filamentphp.com/docs/3.x/panels/dashboard#customizing-the-widgets-grid

a) But I need to remove values/scales at x and y-coordinates :
[![enter image description here][1]][1]

as they have no sense in this case

b) When I put mouse over elements of doughnut - it shows label and value in popup hint
If there is a way alwasys to show these label and value for any element

    "laravel/framework": "^10.28.0",
    "filament/filament": "^3.0-stable",

Thanks in advance!

  [1]: https://i.stack.imgur.com/BQEky.png

**Comment by sinnbeck** [comment]
You can do this to hide the scales
```php
    protected function getOptions(): array|\Filament\Support\RawJs|null
    {
        return [
            'scales' => [
                'x' => ['display' => false],
                'y' => ['display' => false]
            ],
        ];
    }

**Reply by alisalehi1380** [reply]
👌

**Comment by PetroGromovo** [comment]
Thanks ! 
a) Is this (and other charts) wrapper to some external libraries 
b) if yes, please reference to it and a way how can I pass parameters from php to to this lib ?

**Reply by sinnbeck** [reply, marked as answer]
Yup. It's just wrapping chartjs

https://filamentphp.com/docs/3.x/widgets/charts

https://www.chartjs.org/docs/latest/

You pass options using the getsOptions method like above
